include/xgboost/c_api.h

```diff
@@ -653,6 +653,8 @@ XGB_DLL int add_client_key(
     uint8_t* signature,
     size_t sig_len);
 
+XGB_DLL int sync_client_key();
```

Comment:
Actually `sync_client_key()` is never used at all. I just left it in here and exposed it to the user in case we needed it further down the line. I could take out all the corresponding `sync_client_key()` code and the cluster-remote-control example would still work. Do you think we should leave it in or take it all out?

Reply:
I don't think we'll need it (see response to your larger comment about the design).